x-pack/plugins/apm/public/components/app/diagnostics/apm_documents_tab.tsx

```diff
@@ -169,7 +192,13 @@ function IntervalDocCount({
   };
 }) {
   if (interval === undefined) {
-    return <>-</>;
+    return (
+      <>
+        {i18n.translate('xpack.apm.intervalDocCount.-Label', {
+          defaultMessage: '-',
+        })}
```

Comment:
I don't think a `-` needs translation.

Do you really need to return a React Fragment? Can't you just return '-'?

Reply:
@cauemarcondes I think all of the i18n examples are auto-generated.

@CoenWarmer Do you think this is related to the eslint plugin you made? Can we make it less aggressive?

Reply:
Yes, that would be this plugin. Thanks for letting me know, I'm already working on some other improvements so I'll take this in as well.